scripts/github_runner/start_bare-metal_runner.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# MIT License
+#
+# Copyright (c) 2020 Nathaniel Tornow and EASE lab
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+if [ -z $URL ] || [ -z $RUNNER_TOKEN ] || [ -z $RUNNER_LABEL ] || [ -z $RUNNER_NAME ] || [ -z $SANDBOX ]; then
+    echo "Parameters missing"
+    exit -1
+fi
+
+sudo apt-get update
+
+cd
+git clone ${URL}
+cd vhive
+./scripts/cloudlab/setup_node.sh ${SANDBOX}
+
+cd
+mkdir actions-runner && cd actions-runner
+
+curl -o actions-runner-linux-x64-2.280.3.tar.gz -L https://github.com/actions/runner/releases/download/v2.280.3/actions-runner-linux-x64-2.280.3.tar.gz
```

Comment:
please make it fetch the latest version, look at how it is done inside the docker runner

Reply:
checkout [this](https://github.com/ease-lab/vhive/blob/2a0979407942294c3073e0c03008b0936bd202d2/scripts/github_runner/setup_runner.sh#L47).

Reply:
Fixed